### DISCUSSION TITLE
【V5】坐标系的范围到底如何计算

### DISCUSSION THREAD

**Discussion by BinghuiXie** [antvis/G2#5399] [Q&A] [answered]
![image](https://github.com/antvis/G2/assets/34343589/7cd5b43d-c875-4825-b755-30420d10ec3b)
坐标系的范围，用上图描述应该是哪一个部分呢？在下图创建坐标系实例的时候，忽略了中间内边距的部分，却把呼吸区域和外边距区域的范围算了进去，没有理解为什么要这样计算
![image](https://github.com/antvis/G2/assets/34343589/f2040de0-78a6-4608-b039-c090de2324e2)
或者说：
![image](https://github.com/antvis/G2/assets/34343589/9b4cab4c-e288-4820-9efe-318561badbcd)
这张图里面的计算逻辑是对的吗？因为很明显可以看出，加了 padding 以后，“坐标系的范围” 和标记绘制区域完全错开了

**Comment by pearmini** [comment, marked as answer]
这里：https://github.com/antvis/G2/issues/5400
